samples/pom.xml

```diff
@@ -45,7 +45,6 @@
   </properties>
 
   <modules>
-    <module>install-without-bom</module>
     <module>snapshot</module>
     <module>snippets</module>
   </modules>
```

Comment:
Why is "install-without-bom" getting removed?

Reply:
We are not adding any pre-1.0.0 libraries to the google-cloud-bom and thus the libraries-bom. Therefore, the snippets module now contains the direct installation method.

Once this library hits 1.0 and we include it in the home, then we'll bring back the libraries-bom samples and tests. The presence of the libraries bom snippet will cause those instructions to appear in the README

Reply:
FYI: @soyrice 

We need to update the library installation section accordingly.